GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -491,9 +491,18 @@ private string RunFastFetch(string args)
             processInfo.RedirectStandardError = true;
             
             ProcessResult result = ProcessHelper.Run(processInfo);
-            result.Output.Contains("Error").ShouldEqual(false, result.Output);
-            result.Errors.ShouldBeEmpty(result.Errors);
-            result.ExitCode.ShouldEqual(0);
+            result.Output.Contains("Error").ShouldEqual(expectError, result.Output);
+            if (expectError)
+            {
+                Assert.IsTrue(result.Errors.Length > 0);

```

Comment:
In our tests we try to use the `Should` syntax whenever possible.  Rather than doing:

`Assert.IsTrue(result.Errors.Length > 0);`

Switch to something like one of these:

`result.Errors.Length.ShouldBeNonEmpty("Expected an error message");`
`result.Errors.Length.ShouldBeAtLeast(1, "Expected an error message");`

Reply:
What would Should be non empty mean for an integer?

Reply:
Doh, that option should have been `result.Errors.ShouldBeNonEmpty("Expected an error message");`